### ISSUE TITLE
In GAP 4.8 beta IsBound throws error when applied to component object

### ISSUE THREAD

**Issue by sebasguts** [gap-system/gap#389]
In GAP 4.8 beta IsBound returns an Error instead of `false` if applied to a component of an object that is not present. Example:
```
gap> S := SymmetricGroup( 2 );
Sym( [ 1 .. 2 ] )
gap> IsBound( S!.Hallo );
Error, Record: '<rec>.Hallo' must have an assigned value
not in any function at line 14 of *stdin*
you can 'return;' after assigning a value
```

**Comment by ChrisJefferson** [comment]
That's bad! Investigating...

**Comment by ChrisJefferson** [comment]
Pull request now testing, thanks for catching this.

**Comment by alex-konovalov** [comment]
Closed - fixed by PR #390.
